sql/src/test/java/io/crate/lucene/LuceneQueryBuilderTest.java

```diff
@@ -192,7 +191,7 @@ public void testRegexQueryFast() throws Exception {
         Reference value = createReference("foo", DataTypes.STRING);
         Literal pattern = Literal.newLiteral(new BytesRef("[a-z]"));
         Query query = convert(whereClause(RegexpMatchOperator.NAME, value, pattern));
-        assertThat(query, instanceOf(RegexpQuery.class));
+        assertThat(query, instanceOf(XConstantScoreQuery.class));
```

Comment:
maybe add another assertion which verifies that there is an RegexpFilter within the query